tests/cupyx_tests/test_time.py

```diff
@@ -0,0 +1,91 @@
+import mock
+import time
+import unittest
+
+import numpy
+
+import cupy
+import cupyx
+
+
+class _CountCall(object):
+
+    def __init__(self, func):
+        self.func = func
+        self.counter = 0
+
+    def __call__(self, *args, **kwargs):
+        self.counter += 1
+        return self.func(*args, **kwargs)
+
+
+class TestRunPerf(unittest.TestCase):
+
+    def test_cpu_routine(self):
+        @_CountCall
+        def f(x, y):
+            time.sleep(0.1)
+            return x + y
+
+        x = cupy.testing.shaped_random((2, 3), cupy, 'int32')
+        y = cupy.testing.shaped_random((2, 3), cupy, 'int32')
+        assert f.counter == 0
+        perf = cupyx.run_perf('test_name_xxx', f, (x, y), n=10, n_warmup=3)
+        assert f.counter == 13
+        assert perf.name == 'test_name_xxx'
+        assert 0.1 <= perf.cpu_times.mean() <= 0.105
```

Comment:
This kind of check is not robust.
`0.1 <= ` is maybe OK, but `<= 0.105` is likely to break.
For the lower bound check, you should perhaps use `.min()` instead of `.mean()`.

Reply:
Fixed.

Reply:
Now fixed.